jerry-core/lit/lit-strings.c

```diff
@@ -231,7 +231,7 @@ convert_code_point_to_low_surrogate (lit_code_point_t code_point) /**< code poin
  *
  * @return higher code_unit of the surrogate pair
  */
-static ecma_char_t
+ecma_char_t
```

Comment:
These are used from the same compilation unit, do we need to make them non-static?